kuncao0228- Added extractText.py and testExtractTest.py to capitalize…

kuncao0228 opened the pull request:
Variable and Equation Extraction

jpfairbanks commented on bin/sir_model_intro.md:
```diff
@@ -0,0 +1,35 @@
+---
```
I think this needs to be in /examples somewhere.

kuncao0228 commented:
Added more tests, provided a output/input directories for the extracted texts, handles headers

kuncao0228 commented:
removed .pyc and added the require.txt, 
added in the csv output and text inputs with extracted texts
sovMain.py runs with an input argument of the dir_path of the extracted text location

crherlihy commented:
Just a note that if hitting the Rosette api becomes restrictive at some point, we can consider using spaCy and/or textacy for pos tagging/extraction.

kuncao0228 commented:
My recent pull request is no longer using rosette just spaCy

crherlihy commented:
@kuncao0228 great! sorry, am getting caught up.
